Helper/OmiseHelper.php

```diff
@@ -20,7 +20,7 @@
 use Omise\Payment\Model\Config\Rabbitlinepay;
 use Omise\Payment\Model\Config\Ocbcpao;
 use Omise\Payment\Model\Config\Grabpay;
-use Omise\Payment\Model\Config\Cc as Config;
```

Comment:
should we still import `Cc` and update `Config` to `Cc` 🤔 
```php
    /**
     * @var array
     */
    private $cardPaymentMethods = [
        Config::CODE,
        CcGooglePay::CODE
    ];

Reply:
@new4762 Nice catch!

Reply:
@new4762 Nice catch. Thanks. I have imported `Cc` for the code and `Config` for the config related functions.